duckduckgo/iOS
# Autofill in browser changes

Task/Issue URL: https://app.asana.com/0/0/1202806483350357/f
Tech Design URL:
CC:

**Description**:
- Removes blur backgrounds
- Introduces a third button type
- Reenables the key icon
- Stops the saving interface and key icons showing if auth isn't available



**Steps to test this PR**:
1. Test saving and autofill backgrounds look correct. Particularly test both manually and automatically expanding the autofill prompt to see if the background does anything weird (Since it used to have screwy logic to change colour, and now should just remain the same)
1. The autofill prompt should have a new button type for other login items than the primary (should now have a border)
1. Check the key icon appears whenever there is a saved login for a site. Check clicking it works, and just generally nothing is weird with the key
1. Disable auth and check that the key doesn't appear, and neither autofill interface appears (both saving and selecting)



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
